crates/accelerate/src/pauli_lindblad_map/qubit_sparse_pauli.rs

```diff
@@ -0,0 +1,2466 @@
+///   :attr:`bit_terms`   :math:`s`    Each of the non-identity single-qubit Pauli operators.  These
+///                                    correspond to the non-identity :math:`A^{(n)}_i` in the list,
+///                                    where the entries are stored in order of increasing :math:`i`
+///                                    first, and in order of increasing :math:`n` within each term.
+///
+///   :attr:`indices`     :math:`s`    The corresponding qubit (:math:`n`) for each of the operators
+///                                    in :attr:`bit_terms`.  :class:`QubitSparsePauli` requires
+///                                    that this list is term-wise sorted, and algorithms can rely
+///                                    on this invariant being upheld.
+///   ==================  ===========  =============================================================
+///
+/// The parameter :math:`s` is the total number of non-identity single-qubit terms.
+///
+/// The scalar item of the :attr:`bit_terms` array is stored as a numeric byte.  The numeric values
+/// are related to the symplectic Pauli representation that :class:`.SparsePauliOp` uses, and are
+/// accessible with named access by an enumeration:
+///
+/// ..
+///     This is documented manually here because the Python-space `Enum` is generated
+///     programmatically from Rust - it'd be _more_ confusing to try and write a docstring somewhere
+///     else in this source file. The use of `autoattribute` is because it pulls in the numeric
+///     value.
+///
+/// .. py:class:: QubitSparsePauli.BitTerm
+///
+///     See :class:`QubitSparsePauliList.BitTerm` - this is a reference to the same class.
+///
+///
+/// Each of the array-like attributes behaves like a Python sequence.  You can index and slice these
+/// with standard :class:`list`-like semantics.  Slicing an attribute returns a Numpy
+/// :class:`~numpy.ndarray` containing a copy of the relevant data with the natural ``dtype`` of the
+/// field; this lets you easily do mathematics on the results, like bitwise operations on
+/// :attr:`bit_terms`.
+///
+/// Construction
+/// ============
+///
+/// :class:`QubitSparsePauli` defines several constructors.  The default constructor will
+/// attempt to delegate to one of the more specific constructors, based on the type of the input.
+/// You can always use the specific constructors to have more control over the construction.
+///
+/// .. _qubit-sparse-pauli-convert-constructors:
+/// .. table:: Construction from other objects
+///
+///   ============================  ================================================================
+///   Method                        Summary
+///   ============================  ================================================================
+///   :meth:`from_label`            Convert a dense string label into a :class:`.QubitSparsePauli`.
```

Comment:
```suggestion
///   :meth:`from_label`            Convert a dense string label into a :class:`~.QubitSparsePauli`.
```
You may need this for the hyperliink to work, not sure though